packages/clay-modal/stories/index.tsx

```diff
@@ -46,6 +46,30 @@ const ModalWithState: React.FunctionComponent<IProps> = ({
 	);
 };
 
+const MyApp: React.FunctionComponent<any> = ({}) => {
+	const [state, disptach] = useContext(Context);
+
+	return (
+		<ClayButton
+			displayType="primary"
+			onClick={() =>
+				disptach &&
```

Comment:
I had a problem with types so I'll have to expose the `TProvider` interface for people can add type to `useContext`.

Reply:
hmm, thats strange. What was the issue in particular?

Reply:
Oh that's weird, now that I've done some testing seems to work fine, just changing it in `createContext<TProvider>([initialState, () => {}])`. This was breaking before, I don't know why ...

Reply:
I wonder if it relates to the tsconfig issues a few people have run into. Or maybe vscode just needed to restart the ts server.

Reply:
> I wonder if it relates to the tsconfig issues a few people have run into. Or maybe vscode just needed to restart the ts server.

I think this was just VSCode, I don't think it's related to the types we were generating wrong for production. Why am I messing with the same package.